api/src/generated/nexus.ts

```diff
@@ -75,6 +75,10 @@ export interface NexusGenObjects {
     name?: string | null; // String
     shortCode?: string | null; // String
   }
+  Counter: { // root type
+    count: string; // String!
+    message: string; // String!
+  }
```

Comment:
Maybe we would then need to remove those generated files from git whatsoever, to avoid update to our code when a module is updated. Is it even possible? (to remove `generated` folder from repo)